source/includes/_payment.md

```diff
@@ -373,14 +418,23 @@ KillBillClient::Model::Payment.find_by_external_key(external_key,
 ```
 
 ```python
-paymentApi = killbill.api.PaymentApi()
+paymentApi = killbill.PaymentApi()
+
 payment_external_key = 'sample_external_key' 
 
-paymentApi.get_payment_by_external_key(payment_external_key,
-                                       api_key,
-                                       api_secret)
+payment = paymentApi.get_payment_by_external_key(payment_external_key)
+```
```

Comment:
Is a new line required after the Python code?

Reply:
Reshma, I think it is showing fine in local. 

![image](https://github.com/killbill/slate/assets/75751774/4fb7d2de-6e91-4d1a-9b3f-f6b7a2b50cfb)